crates/cli/src/subcommands/logs.rs

```diff
@@ -17,16 +18,9 @@ pub fn cli() -> clap::Command {
                 .required(true)
                 .help("The domain or address of the database to print logs from"),
         )
+        .arg(common_args::server())
         .arg(
-            Arg::new("server")
-                .long("server")
-                .short('s')
-                .help("The nickname, host name or URL of the server hosting the database"),
```

Comment:
note that this helptext changed by using the `common_args` version. I don't personally feel that this "custom" version is meaningfully more helpful.